source/_posts/2017/2017-01-04-why-I-deleted-my-linkedin-account.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: "Why I deleted my LinkedIn account"
+perex: '''
+    I used LinkedIn since 2008 when it was sending spam to all my friends. After 8 years of tuning my profile to 95 % score, reading mostly copy-pasted messages,
+    checking false notifications, flirting with HR girls and hoping for up votes for my recent skills, <strong>I've decided it is time to let it go</strong>. Why and what I'm gonna do now?
+'''
+lang: en
+---
+
+*Disclaimer: even if I get few abroad offers, my experience is related to work environment in the Czech Republic.*
+
+I was reviewing my 2016 on social networks disconnection: I reduced 350 friends on Facebook to 20, unfollowed all but 2 groups and I dropped all people I follow on Twitter. 
+My attention got better and I could focus on things that really mattered to me. That's why I started though about the next step.
+
+And LinkedIn account was something scratching on my mind for a long time.
+
+So I asked myself?
+
+
+## What is LinkedIn good for me?
+
+- It helped me in my first 3 years in paid PHP programming field. 
+- I got some references and up-votes for my skills, I attended university so I thought it gives me credibility.
+- It gave me self-confidence when talking about money.
+- Everybody did it, so did I (like that time I jumped out of the window :).
+
+
+## Where LinkedIn Fails for me  
+
+### It's kinda Broken and Stuck Service
+
+I was getting fake notifications of new connections. I got over 20 pings for 6 moths-old and answered message.
+Also, I didn't notice many changes between 2008 and 2016. It could be much more further. Btw, do you remember MySpace?
+ 
+
+### Irrelevant Messages - 95 % Job Offers are Spam 
+
+I only got **2 jobs I wanted** in the whole 8 years. Most of offers are general random messages from bots or people,
```

Comment:
Well I got many offers I wanted, bot only 2 of them were both great and realized.